### PR TITLE
drop goose

### PR DESCRIPTION
**Pull request by dgoodwin**
- Drop use of goose for sql schema management.
- Specify explicit columns in job_results function.
- Remove goose from the sippy container.

[TRT-329](https://issues.redhat.com//browse/TRT-329)

### CODE REVIEW

File: pkg/db/functions.go [hunk 1]
```diff
@@ -96,7 +96,25 @@ WITH results AS (
...
+       current_infra_fails,
+       id,
+       created_at,
+       updated_at,
+       deleted_at,
+       name,
+       release,
+       variants,
+       test_grid_url,
+       kind,
```
Comment (dgoodwin) [inline pkg/db/functions.go:117]: This was interesting, it broke on fresh db's once I switched back to gorm automigrate.

**Comment by dgoodwin** [comment]
/hold 

Depends on https://github.com/openshift/continuous-release-jobs/pull/1162

File: Dockerfile [hunk 2]
```diff
@@ -13,7 +12,6 @@ COPY --from=builder /go/src/sippy/scripts/fetchdata.sh /bin/fetchdata.sh
 COPY --from=builder /go/src/sippy/scripts/fetchdata-prow.sh /bin/fetchdata-prow.sh
 COPY --from=builder /go/src/sippy/scripts/fetchdata-kube.sh /bin/fetchdata-kube.sh
 COPY --from=builder /go/src/sippy/historical-data /historical-data/
-COPY --from=builder /go/bin/goose /bin/goose
 COPY ./dbmigration /usr/share/sippydbmigrations
```
Comment (stbenjam) [inline Dockerfile:15]: This one needs to go too

Comment (dgoodwin) [reply Dockerfile]: Thanks, updated.

**Pull request merged**